data-prepper-logstash-configuration/src/test/java/org/opensearch/dataprepper/logstash/mapping/TestDataProvider.java

```diff
@@ -37,14 +64,14 @@ public static LogstashPlugin pluginData() {
                 .attributes(Arrays.asList(getArrayTypeAttribute(), getStringTypeAttribute())).build();
     }
 
-    public static PluginModel getSamplePluginModel() {
-        Map<String, Object> attributes = new LinkedHashMap<>();
-        attributes.put("aws_sigv4", true);
-        attributes.put("insecure", false);
-        attributes.put("aws_region", "us-west-2");
-        attributes.put("hosts", Collections.singletonList("https://localhost:9200"));
-        return new PluginModel("opensearch", attributes);
-    }
+//    public static PluginModel getSamplePluginModel() {
```

Comment:
I'd rather follow the approach of removing code than commenting it out. Git history can help us get back if we want.

Reply:
Yeah, I actually forgot to remove it.